openshift/cron-chips.yml

```diff
@@ -0,0 +1,203 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+labels:
+  app: ${APP_NAME}
+parameters:
+  - name: APP_NAME
+    description: App name that OpenShift objects will be prefixed with
+    required: true
+  - name: ENV
+    description: Environment name
+    value: dev
+    required: true
+  - name: IMAGE_NAMESPACE
+    description: Namespace from which to pull the image
+    required: true
+  - name: IMAGE_TAG
+    description: Image tag from image stream
+    value: latest
+    required: true
+  - name: CONFIG_VERSION
+    description: The github repository version of Config (Commit SHA)
+    value: default
+objects:
+  - apiVersion: batch/v1
+    kind: CronJob
+    metadata:
+      name: ${APP_NAME}-cronjob-chips
+    spec:
+      schedule: 0 0 0 0 0
+      concurrencyPolicy: 'Replace'
+      startingDeadlineSeconds: 10
+      successfulJobsHistoryLimit: 1
```

Comment:
Since these pods won't be counted in our long time compute quota we might want to keep 3 for logs?

Reply:
(Just since this one is nightly - I'm not sure we retain the logs otherwise)

Reply:
I'm okay with that